sys/bindgen.rs

```diff
@@ -3,11 +3,16 @@ use std::{env, path::Path};
 #[derive(Debug)]
 struct Renamer;
 impl bindgen::callbacks::ParseCallbacks for Renamer {
-    fn item_name(&self, name: &str) -> Option<String> {
-        // Remove ffx/ffxfsr2 prefixes.
-        let name = name.replace("ffx", "").replace("Ffx", "");
-
-        Some(name)
+    fn item_name(&self, item_info: bindgen::callbacks::ItemInfo) -> Option<String> {
+        // Remove ffx prefixes
+        match item_info.kind {
+            bindgen::callbacks::ItemKind::Module => None,
+            bindgen::callbacks::ItemKind::Type => item_info.name.strip_prefix("Ffx"),
+            bindgen::callbacks::ItemKind::Function => item_info.name.strip_prefix("ffx"),
+            bindgen::callbacks::ItemKind::Var => item_info.name.strip_prefix("FFX_"),
+            _ => None,
+        }
+        .map(str::to_owned)
```

Comment:
In the future we could follow up on the original comment and also strip repetitive `component` prefixes, that is especially trivial after #35.